src/backends/torch/torchlib.cc

```diff
@@ -1467,8 +1487,12 @@ namespace dd
                     output = torch::stack(outputs);
                   }
 
-                if (extract_layer.empty() && _classification && !_timeserie)
+                // XXX: why is (!_timeserie) needed here? Aren't _timeserie and
+                // _classification mutually exclusive?
```

Comment:
timeseries could be classified (but  we never had such use case)

Reply:
Will add a comment